seahorse/lib/seahorse/number_helper.rb

```diff
@@ -1,14 +1,12 @@
 # frozen_string_literal: true
 
-require 'time'
-
 module Seahorse
-  # A module that provides helper methods to convert from Time objects to
+  # A module that provides helper methods to convert Numeric objects to
   # protocol specific serializable formats.
   # @api private
   module NumberHelper
     class << self
-      # @param [Number,String] n
+      # @param [Number,String] input
       # @return [String] The serialized number
       def serialize(input)
         if input == ::Float::INFINITY then 'Infinity'
```

Comment:
Should these instead be case statements? You might be able to check ::Float::NAN instead of calling .nan?

Reply:
I did have this as a case statement, but rubocop got mad at me for this because it's not straightforward comparisions. 
 And Nan is never equal to nan, so that check doesn't work.